service/core/SoapHelperWebService.php

```diff
@@ -173,7 +173,7 @@ function validate_user($user_name, $password){
 
 		$GLOBALS['log']->info('End: SoapHelperWebServices->validate_user - validation passed');
 		return true;
-	}else if(function_exists('mcrypt_cbc')){
+	}else if(function_exists('openssl_decrypt')){
```

Comment:
https://www.php-fig.org/psr/psr-2/#51-if-elseif-else

Reply:
I have reformatted the files to meet PSR-2.